tests/model/generated/test_api_context.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+from bunq.sdk.context import ApiContext
+from bunq.sdk.json import converter
+from tests.bunq_test import BunqSdkTestCase
+
+
+class ApiContextTest(BunqSdkTestCase):
+    """
+    Tests:
+        ApiContext
+    """
+
+    _TMP_FILE_PATH = '/context-save-test.conf'
+
+    @classmethod
+    def setUpClass(cls):
+        cls._FILE_MODE_READ = ApiContext._FILE_MODE_READ
+        cls._API_CONTEXT = cls._get_api_context()
+        cls._TMP_FILE_PATH_FULL = (cls._get_directory_test_root() +
+                                   cls._TMP_FILE_PATH)
+
+    def test_api_context_save(self):
+        """
+        Converts an ApiContext to JSON data, saves the same ApiContext to a
+        temporary file, and compares whether the JSON data is equal to the
+        data in the file.
+
+        Removes the temporary file before assertion.
+        """
+
+        context_json = converter.class_to_json(self._API_CONTEXT)
+
+        self._API_CONTEXT.save(self._TMP_FILE_PATH_FULL)
+
+        with open(self._TMP_FILE_PATH_FULL, self._FILE_MODE_READ) as file:
```

Comment:
`file` is a builtin python please append a `_` at the end of it or rename the variable.